content/en/docs/Getting Started/_index.md

```diff
@@ -17,16 +17,17 @@ Tekton](/docs/getting-started/tasks/). If you prefer, watch the following
 videos to learn the basics of how Tekton works before your first hands-on
 experience:
 
```

Comment:
Having the larger containers for the videos was useful. With this updated small size many of the controls are hidden / not easily accessible, chapter previews render off the top of the video container, and of course the video itself is only really watchable by switching to fullscreen or by clicking the link to open on YouTube.

Reply:
Yeah, I wasn't sure about this one. Thanks for the feedback. I'm keeping the cards' as standalone instead of the card pane.